README.rst

```diff
@@ -59,6 +59,33 @@ There is one special method which allows transfer file between hosts.
         h2, "/path/to/file/on/h2/or/target/dir",
     )
 
+You can also mount devices.
+
+.. code:: python
+
+    from rrmngmnt import Host, RootUser
```

Comment:
get rid of creating instance of Host and adding user, all examples already assumes that you have 'h' variable containing Host instance ... see examples above